internal/api/instrumentation/probes.go

```diff
@@ -149,13 +151,13 @@ func Start() {
 	validationFailureTotal.WithLabelValues(labelTenantAnemic)
 	validationFailureTotal.WithLabelValues(labelSatellite)
 
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunCreate, LabelAnsibleRequest)
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunHostCreate, LabelAnsibleRequest)
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunRead, LabelAnsibleRequest)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunCreate, LabelAnsibleRequest, ApiVersion)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunHostCreate, LabelAnsibleRequest, ApiVersion)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunRead, LabelAnsibleRequest, ApiVersion)
 
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunCreate, LabelSatRequest)
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunHostCreate, LabelSatRequest)
-	errorTotal.WithLabelValues(labelDb, labelPlaybookRunRead, LabelSatRequest)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunCreate, LabelSatRequest, ApiVersion)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunHostCreate, LabelSatRequest, ApiVersion)
+	errorTotal.WithLabelValues(labelDb, labelPlaybookRunRead, LabelSatRequest, ApiVersion)
```

Comment:
@aleccohan I still don't fully understand metrics configurations, but it looks like you're only initializing `v1` api_version metrics and not `v2`. Is that correct? If so, should we also initialize `v2` configs here?

Reply:
@tahmidefaz great question, these initializations are just for adding the initial value, at the time of each actual metric call the API version is set to what it's supposed to be

Reply:
Got it! Then, it looks good to me. :+1: